src/collections/map-of-the-week/mon-mar-18-2024-13-00-00-gmt-0600-central-standard-time.md

```diff
@@ -0,0 +1,5 @@
+---
+mapId: 3a809
+review: The top diff has a lot of energy, variety in patterns, and absolutely represents the spirit of Balanced maps with moderately fast patterns, some body movement, and engaging patterns. The two downmaps provide a nice progression that will still prove challenging and intense to less experienced players.
+startDate: 2024-03-11T19:00:00.000Z

```

Comment:
The date seems wrong. Shouldn't it be 2024-03-18?

Reply:
Good catch! Yes! Will fix